decidim-budgets/app/models/decidim/budgets/project.rb

```diff
@@ -36,6 +37,11 @@ def comments_have_votes?
       def confirmed_orders_count
         orders.finished.count
       end
+
+      # Public: Overrides the `notifiable?` Notifiable concern method.
+      def notifiable?(_context)
+        false
```

Comment:
I find this confusing too. What I interpret `Notifiable` as a module means that the including object needs to define the interface (`#notifiable?` and `#users_to_notify`), whereas `notifiable?` itself means whether the class is actually notifiable or not. Is that it?

Also, should this inclusion be moved to the `Commentable`'s module?